modules/FilterMutectCalls.nf

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env nextflow
+
+
+nextflow.enable.dsl=2
+
+
+process FilterMutectCalls {
+
+        tag "FilterMutectCalls $bam_id"
+        publishDir "${params.outDir}", mode:'copy'
+
+
+
+        input :
+		tuple val(bam_id) , file(bam_N), file(bam_T)
+                path pair_unfiltered_stats
+                path pair_contaminationTable
+                path segments_T_table
+                path pair_unfiltered_vcf
+                path pair_unfiltered_vcf_index
+                path pair_read_orientation_model
+                path(ref)
+                
+
+        output :
+                path ("${bam_id}-T_${bam_id}-N.filtered.vcf.gz")
+
+        shell:
```

Comment:
Since we are using the --java-options "-Xmx16g" inside the bash block,

When using " inside the bash script block, we need to use

shell with ''' and not script with """
To recognise the "-Xmx16g"